galaxy_importer/ansible_test/runners/openshift_job.py

```diff
@@ -16,12 +16,145 @@
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
 
-# from galaxy_importer.ansible_test import container_build
+import os
+import requests
+import time
+import uuid
+import yaml
+
+from galaxy_importer import config
+from galaxy_importer import exceptions
 from galaxy_importer.ansible_test.runners.base import BaseTestRunner
 
 
+cfg = config.Config()
+POD_CHECK_RETRIES = 120  # TODO: try to shorten once not pulling image from quay
+POD_CHECK_DELAY_SECONDS = 1
+OCP_TOKEN_PATH = '/var/run/secrets/kubernetes.io/serviceaccount/token'
+
+
 class OpenshiftJobTestRunner(BaseTestRunner):
     """Run image as an openshift job."""
     def run(self):
+        # TODO: build image with pulp-container when ready
         # image = container_build.build_image_with_artifact()
-        pass
+        image = 'quay.io/awcrosby/ans-test-with-archive'
+
+        filename = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'job_template.yaml')
```

Comment:
Better to use `pkg_resources` for resolving data file path within python package.

Reply:
updated